server/events/comment_parser_test.go

```diff
@@ -261,7 +261,6 @@ func TestParse_InvalidCommand(t *testing.T) {
 		"a warning.")
 	comments := []string{
 		"atlantis paln",
-		"atlantis Plan",
```

Comment:
Can you add this to the valid command test?

Reply:
I didn't see any valid command test so I created a new one (I hope I didn't miss anything). Maybe I can create a `TestParse_ValidCommand` and see if maybe I can merge the `TestParse_Multiline` in that one too.

Reply:
Do you think that looks better?

Reply:
Yes thank you